src/test/java/pages/CartPage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import core.utils.ItemUtils;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CartPage extends BasePage {
+
+    @FindBy(xpath = "//span[@id='sc-subtotal-amount-activecart']/span[@class='a-size-medium a-color-base sc-price sc-white-space-nowrap']")
+    private WebElement subtotalPrice;
+
+    @FindBy(xpath = "//span[@id='sc-subtotal-label-activecart']") //Subtotal (2 items):
```

Comment:
what this comment should give to the reader?

Reply:
forgot to delete it